frontend/packages/core/src/Feedback/error/details.tsx

```diff
@@ -0,0 +1,186 @@
+    padding: "12px 16px 12px 24px",
+    minHeight: "fit-content",
+    "& .MuiAccordionSummary-content": {
+      margin: "0",
+      alignItems: "center",
+    },
+    "&.MuiAccordionSummary-root.Mui-expanded": {
+      minHeight: "unset",
+    },
+  },
+  props => ({
+    borderBottomLeftRadius: props["data-expanded"] ? "0" : "8px",
+    borderBottomRightRadius: props["data-expanded"] ? "0" : "8px",
+  })
+);
+
+const AccordionDetails = styled(MuiAccordionDetails)({
+  background: "linear-gradient(to right, #DB3615 8px, #FFFFFF 0%)",
+  padding: "0",
+  paddingLeft: "8px",
+  borderBottomLeftRadius: "8px",
+  borderBottomRightRadius: "8px",
+  display: "flex",
+  flexDirection: "column",
+  // boxShadow: "0px 4px 6px rgba(53, 72, 212, 0.2)",
+});
+
+const ListItem = styled.li({
+  "::marker": {
+    color: "rgba(13, 16, 48, 0.6)",
+  },
+  margin: "2px 0",
+});
+
+const ErrorDetailContainer = styled.div({
+  width: "100%",
+  border: "1px solid #E7E7EA",
+  padding: "16px 16px 16px 24px",
+  borderBottomRightRadius: "8px",
+  borderTop: "unset",
+});
+
+const ErrorDetailText = styled.div({
+  color: "rgba(13, 16, 48, 0.6)",
+  fontSize: "14px",
+});
+
+const DialogButton = styled(Button)({
+  color: "#3548D4",
+  fontWeight: 700,
+  fontSize: "14px",
+  padding: "9px 32px",
+});
+
+interface ErrorDetailsProps {
+  error: ClutchError;
+}
+
+const ErrorDetails = ({ error }: ErrorDetailsProps) => {
+  const [detailsOpen, setDetailsOpen] = React.useState(false);
+  const [expanded, setExpanded] = useControlled({
+    controlled: undefined,
+    default: false,
+    name: "Error Accordion",
+    state: "expanded",
+  });
+
+  React.useEffect(() => {
+    setDetailsOpen(false);
+  }, [error]);
+
+  const hasWrappedErrorDetails =
+    error.details.filter(detail => isClutchErrorDetails(detail)).length > 0;
+
+  const summaryIconStyle = { marginRight: "8px" };
+
+  return (
+    <>
+      <ErrorDetailDivider />
+      <Accordion elevation={0} expanded={expanded}>
+        <AccordionSummary
+          aria-controls="panel1a-content"
+          data-expanded={expanded}
+          onClick={() => setExpanded(!expanded)}
+        >
+          {!expanded ? (
+            <>
+              <ChevronRightIcon style={summaryIconStyle} /> Show more
+            </>
+          ) : (
+            <>
+              <KeyboardArrowDownIcon style={summaryIconStyle} /> Show less
+            </>
+          )}
+        </AccordionSummary>
+        <AccordionDetails>
+          <ErrorDetailContainer>
+            {hasWrappedErrorDetails && (
+              <div>
+                <ErrorDetailText style={{ color: "#0D1030" }}>
+                  The following errors were encountered:
+                </ErrorDetailText>
+                <ul>
+                  {error.details.map(detail => {
+                    // Only render Clutch Error wrapped details errors here
+                    if (isClutchErrorDetails(detail)) {
+                      const renderMax = 2;
```

Comment:
nit: make global const at top of file somewhere. also i would increase it to somewhere between 3 and 5